src/components/DetailsCard/cardProps.ts

```diff
@@ -0,0 +1,19 @@
+export type SocialMediaNames = 'facebook' | 'twitter' | 'instagram' | 'github';
```

Comment:
It could be only one name at a time so use a singular form `SocialMediaName`.

Reply:
Changed ✅